sklearn/svm/src/liblinear/liblinear_helper.c

```diff
@@ -163,18 +157,11 @@ struct problem * csr_set_problem (char *X, int double_precision_X,
     if (problem == NULL) return NULL;
     problem->l = n_samples;
     problem->sample_weight = (double *) sample_weight;
```

Comment:
Cosmetic: Move this down so that the fields are set in the same order in set_problem and csr_set_problem...

Reply:
Done.